package.json

```diff
@@ -23,6 +23,7 @@
     "@types/react-dom": "^17.0.9",
     "@types/react-google-recaptcha": "^2.1.5",
     "@types/react-modal": "^3.12.1",
+    "@typescript-eslint/scope-manager": "5.57.0",
```

Comment:
@sarazaToussaint Why have you installed this new package.?

Reply:
@sarazaToussaint You haven't explained this. Could you please explain this package's usage.

Reply:
@anshgoyalevil thank you for reminding me, It was installed by mistake. I already uninstall it